index.d.ts

```diff
@@ -1,5 +1,9 @@
-declare function anySignal(signals: AbortSignal[]): AbortSignal;
+interface CompositeSignal extends AbortSignal {
```

Comment:
I get that this is a signal composed of a lot of other signals hence the name, but that's really an internal implementation detail - the addition here is a `.clear` method so maybe this is a little more self-descriptive?

```suggestion
interface ClearableSignal extends AbortSignal {
```